Templating/templates/swift.methods.mustache

```diff
@@ -1,12 +1,12 @@
     // MARK: - {{ name }}
     
-{{# isMac }}{{# icons }}    class func {{ frameworkPrefix }}{{ method }}() -> {{ imageClass }}? {
-        return {{ imageClass }}(named:{{ constantStructName }}.{{ method }})
+{{# isMac }}{{# icons }}    class func {{ frameworkPrefix }}{{ method }}() -> {{ imageClass }} {
```

Comment:
How about making these `static` instead of `class` functions?  They shouldn't ever be overridden, yes?